pyproject.toml

```diff
@@ -21,6 +21,9 @@ classifiers = [ # // list of PyPI trove classifiers to describe the project
 repository = "https://github.com/NRLMMD-GEOIPS/@package@"
 #documentation = "set the url"
 include = ["**/*.txt", "**/*.py", "**/*.yaml"]
+# @ Update with your package name, ie:
+# @ packages = [{include = "@package@"}]  # package available at the root of the project
+packages = [{include = "my_package"}]
```

Comment:
Should this be `packages = [{include = "@package@}]` for the auto-replace?

Reply:
I wasn't sure.  There are a few other places in this file where we use "my_package" and have "@package@" in a comment, so I followed that standard.